.env.example

```
@@ -73,6 +73,10 @@ VITE_PUBLIC_API_URL=http://localhost:3001
 VITE_PUBLIC_APP_NAME=twotter
 # Databuddy analytics client ID (from https://app.databuddy.cc). Leave blank to disable.
 VITE_PUBLIC_DATABUDDY_CLIENT_ID=
+# Databuddy server-side API key (from https://app.databuddy.cc, format: dbdy_xxx). Leave blank to disable.
+DATABUDDY_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DATABUDDY_API_KEY key should go before the VITE_PUBLIC_API_URL key

(UnorderedKey)